rubix-tests/src/test/robotframework/shared/docker/create_docker_compose.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+import sys
+
+master_str = "version: '3.7'\n" \
+      "networks:\n" \
+      "  default:\n" \
+      "    external:\n" \
+      "      name: network-rubix-build\n" \
+      "services:\n" \
+      "  rubix-master:\n" \
+      "    build:\n" \
+      "      context: .\n" \
+      "      args:\n" \
+      "        is_master: \"true\"\n" \
+      "    volumes:\n" \
+      "      - /tmp/rubix/tests:/tmp/rubix/tests\n" \
+      "      - /tmp/rubix/jars:/usr/lib/rubix/lib\n" \
+      "    networks:\n" \
+      "      default:\n" \
+      "        ipv4_address: 172.18.8.0\n"
+
+worker_str= ":  \n" \
```

Comment:
Can we make this a format string instead of concatenation to improve clarity?
Ex:
```
worker_str = """
{}:
...
```
```
worker_str.format(service_name)